spec/aptible/cli/helpers/tunnel_spec.rb

```diff
@@ -35,13 +35,13 @@
   end
 
   it 'captures and displays port discovery errors' do
-    helper = described_class.new({ 'FAIL_PORT' => '1' }, ['ssh_mock.rb'])
+    helper = described_class.new({ 'FAIL_PORT' => '1' }, ['ssh'])
     expect { helper.start }
       .to raise_error(/Failed to request.*Something went wrong/m)
   end
 
   it 'captures and displays tunnel errors' do
-    helper = described_class.new({ 'FAIL_TUNNEL' => '1' }, ['ssh_mock.rb'])
+    helper = described_class.new({ 'FAIL_TUNNEL' => '1' }, ['ssh'])
```

Comment:
> Out of curiosity, it is not possible to coerce Windows into running the Ruby file as a program directly? Do we have to use a wrapper binary here? (I don't mind if we have to, though!)

I haven't managed to get it to work without the wrapper binary on my VM, even though I had defined that`.rb` files should be opened by ruby.exe. I've tried specifying the full path, adding `.rb` to `%PATHEXT%`, and nothing has worked. 

The only other option that I did get working was prepending the args array with `'ruby'` and specifying the full path to the `ssh_mock.rb` file. 
 - 

I suspect it has something to do with `%PATHEXT%`